slices/libpam-runtime.yaml

```diff
@@ -18,40 +18,133 @@ slices:
   # pam-config and pam-defaults slices are only used for generation, and are useless
   # for anything else.
 
-  # Applications that need libpam for now should just rely on the config and var slices for now -
-  # until appropriate solution for emulating what pam-auth-update can do (or maybe something else).
+  # Emulate in part what pam-auth-update does. There is a short coming right now, we
+  # cannot include other files from other slices that install into
+  # /usr/share/pam-configs/*
+  # since this mutation script wont be able to access them as of writing.
```

Comment:
```suggestion
  # Emulate in part what pam-auth-update does. There is a short coming right now, we
  # cannot include other files from other slices that install into
  #   /usr/share/pam-configs/*
  # since this mutation script wont be able to access them as of writing.
```